public/index.html

```diff
@@ -60,9 +79,35 @@
           password: $('#login-form-password').val(),
         }, function(res) {
           if (res.err) return error(res.err);
-          $('#message').html(messageTmpl.render({message: `Welcome ${res.data.name}!`}));
+          $('#account').html(accountTmpl.render({
+            data: res.data,
```

Comment:
Please expand this data so that it is clear that what you are rendering.

Reply:
What do you mean by expanding data?
should I put the object {data:res.data} into another variable and import it here?